extensions/sql-assessment/src/tabs/assessmentMainTab.ts

```diff
@@ -96,8 +95,14 @@ export class SqlAssessmentMainTab extends SqlAssessmentTab {
 		];
 	}
 
-	private async createToolbar(view: azdata.ModelView): Promise<azdata.ToolbarContainer> {
+	private limitLongName(name: string): string {
+		if (name.length > 40) {
```

Comment:
Normally for this you'd just use the text overflow. Manually changing the string like this isn't great for screen readers.

Although I'm wondering where this is even necessary - the only strings that use this are all < 40 characters.

Reply:
I spent a lot of time but wasn't able to accomplish this by css styles, my custom styles were not applied to toolbar item.

Reply:
Hm, ok. Might be an issue with the core but not something I'm going to be able to investigate right now so this is fine.